src/calendar/view/CalendarView.ts

```diff
@@ -376,7 +378,7 @@ export class CalendarView extends BaseTopLevelView implements TopLevelView<Calen
 		}
 
 		Promise.all([calendarInfos, locator.mailModel.getUserMailboxDetails()]).then(([calendars, mailboxDetails]) =>
-			showCalendarEventDialog(dateToUse, calendars, mailboxDetails),
+			showCalendarEventDialog(assertNotNull(date, "date for new event was null"), calendars, mailboxDetails),
```

Comment:
we can really do without assert here, maybe make a new variable?

Reply:
tried that, TS was not happy (check the compare button next to the force push)